chromium_src/chrome/browser/about_flags.cc

```diff
@@ -0,0 +1,39 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#define SetFeatureEntryEnabled SetFeatureEntryEnabled_ChromiumImpl
+#include "../../../../chrome/browser/about_flags.cc"  // NOLINT
```

Comment:
Needed because `SetFeatureEntryEnabled` in about_flags.cc calls, `SetFeatureEnabled` in flags_state.cc